JulianArroyo/my_app
# feature/6-install-test-pack

- Installation of symfony/test-pack in --dev environment
- Change route of /health-check endpoint to /api/v1/health-check
- Add functional test to /api/v1/health-check endpoint
